### ISSUE TITLE
Add AQI information to weather (or create: aqi?)

### ISSUE THREAD

**Issue by ethanhs** [ocf/ircbot#182]
![image](https://user-images.githubusercontent.com/9504279/92323428-807a2e80-efed-11ea-8704-5aa3bd2a84d4.png)

https://airnowapi.org provides API access to AQI data.

I think it would be nice to include this information in weather, but perhaps a separate command would be better? I have no strong feeling.

You can sign up for an account and you get a GUID(?) API key.

Calls look like:
```
http://www.airnowapi.org/aq/observation/zipCode/current/?format=application/json&zipCode=94720&distance=25&API_KEY=<REDACTED>
```
The output changes based on region (ugh) but gives either the catagory (Unhealthy/Moderate/etc) or the AQI value.

You can also get data based on lat/lon information.

**Comment by ja5087** [comment]
PurpleAir was also considered a reputable source through the past few wildfire seasons. They have a lot of sensors. Of course any of them should be fine.

**Comment by Boomaa23** [comment]
Closed by #200, added to `weather`.
